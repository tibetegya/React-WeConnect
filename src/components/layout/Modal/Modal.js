import React from 'react'
import PropTypes from 'prop-types'

const Modal = props => {
  return (
    <div>
      
    </div>
  )
}

Modal.propTypes = {

}

export default Modal
