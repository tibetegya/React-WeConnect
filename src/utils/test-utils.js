import React from 'react';
import PropTypes from 'prop-types';
import { render } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import { Provider as ReduxProvider } from 'react-redux';
import store from '../redux/store';
import { defaultTheme } from './themes';

const AllTheProviders = ({ children }) => (
  <ReduxProvider store={store}>
    <ThemeProvider theme={defaultTheme}>
      {children}
    </ThemeProvider>
  </ReduxProvider>
);

AllTheProviders.propTypes = {
  children: PropTypes.node.isRequired,
};

const customRender = (ui, options) => render(ui, { wrapper: AllTheProviders, ...options });

// re-export everything
export * from '@testing-library/react';

// override render method
export { customRender as render };
