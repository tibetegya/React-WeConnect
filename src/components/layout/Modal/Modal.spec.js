import React from 'react';
import { render } from 'test-utils';
import Modal from './Modal';

describe('Modal Component', () => {
  it('should render corectly', () => {
    const { getByTestId } = render(<Modal />);
    expect(getByTestId('modal')).toMatchSnapshot();
  });
});
