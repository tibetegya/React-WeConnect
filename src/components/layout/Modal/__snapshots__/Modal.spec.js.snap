// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Component should render corectly 1`] = `
<div
  data-testid="modal"
>
  Modal
</div>
`;
